### PR TITLE
feat: change default font to Inter and Menlo

### PR DESCRIPTION
**Pull request by saihaj**

### CODE REVIEW

**Review by n1ru4l — APPROVED** [review]
✅ ✅ ✅ ✅ ✅ ✅

**Review by notrab — APPROVED** [review]
:shipit:

File: packages/cli/src/nextConfig.ts [hunk 1]
```diff
@@ -453,7 +453,10 @@ export async function writeDocument() {
             <Head />
             <head>
               <link rel="preconnect" href="https://fonts.gstatic.com" />
-              <link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap" rel="stylesheet" />
+              <link
+                href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&display=swap"
+                rel="stylesheet"
+              ></link>
```
Comment (B2o5T) [inline packages/cli/src/nextConfig.ts:459]:
```suggestion
              <link
                href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&display=swap"
                rel="stylesheet"
              />
```

**Review by B2o5T — APPROVED** [review]

File: examples/basic/src/pages/_document.tsx [hunk 2]
```diff
@@ -9,7 +9,10 @@ export default class MyDocument extends Document {
         <Head />
         <head>
           <link rel="preconnect" href="https://fonts.gstatic.com" />
-          <link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap" rel="stylesheet" />
+          <link
+            href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&display=swap"
+            rel="stylesheet"
+          ></link>
```
Comment (B2o5T) [inline examples/basic/src/pages/_document.tsx:15]:
```suggestion
          <link
            href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&display=swap"
            rel="stylesheet"
          />
```

**Pull request merged**